### ISSUE TITLE
How can I use the code to detect multi objects?

### ISSUE THREAD

**Issue by kanta0504** [varunagrawal/tiny-faces-pytorch#32]
Hi,

Thank you for your code, I am thankful for your time and efforts. 

I am trying your code to detect multi objects.
I create a custom dataset and change "num_objects" to multi number.
But, I get the error at line 81 in loss.py because of the difference in the shapes of "regression" and "regression_map".

Could you please confirm the code to detect the multi objects?

Thank you for your time and effort, I really appreciate it!

Regards,

**Comment by varunagrawal** [comment]
I am not certain what you are doing here. It seems like your dataloader is not computing the ground truth regression tensor correctly which is what is causing the issue. Or maybe it is, and the model definition needs to be updated. Without access to either your code or dataset, it is pretty much impossible for me to debug this issue.
